tests/test_mark_tokens.py

```diff
@@ -814,6 +819,10 @@ def assert_nodes_equal(self, t1, t2):
     else:
       self.assertEqual(type(t1), type(t2))
 
+    if isinstance(t1, AstroidPosition):
```

Comment:
Just checking that this is a separate kind of node and not a base class for everything in new astroid :)

Reply:
It's not a node, it's this:

```python
class Position(NamedTuple):
    """Position with line and column information."""

    lineno: int
    col_offset: int
    end_lineno: int
    end_col_offset: int
```